apps/coscrad-frontend/src/utils/generic-components/presenters/tables/generic-index-table-presenter/index-table.tsx

```diff
@@ -284,9 +316,9 @@ export const IndexTable = <T extends IBaseViewModel>({
                 ) : (
                     <p>Click to enable input method: {simulatedKeyboard.name}</p>
                 )}
-            </div>
+            </Box>
 
-            <div className="records-table">{table}</div>
-        </div>
+            <Box>{table}</Box>
+        </Stack>
     );
```

Comment:
This file is way too long. We have a story to refactor all of this, but I haven't gotten to it yet.

Reply:
At this stage I could easily break out some sub-components, would it be worth doing at least that?

Reply:
Won't do this now.